### PR TITLE
Iops, throughput and instance view

### PR DESCRIPTION
**Pull request by robhat1**
Added IOPS and BW properties on managed disk  for UltraSSD disks.
These properties can be set only on the VMScaleSet but are also returned via VMSCaleSet VMs.
ListVMs at susbcription level now supports a statusOnly=true query param and CRP would returned a trimmed down instance view in that case
Added samples for GETVM and GETVMInstanceView

### Latest improvements:
<i>MSFT employees can try out our new experience at <b>[OpenAPI Hub](https://aka.ms/openapiportal) </b> - one location for using our validation tools and finding your workflow. 
</i><br>
### Contribution checklist:
- [ ] I have reviewed the [documentation](https://github.com/Azure/azure-rest-api-specs#basics) for the workflow.
- [ ] [Validation tools](https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/swagger-checklist.md#validation-tools-for-swagger-checklist) were run on swagger spec(s) and have all been fixed in this PR.
- [ ] The [OpenAPI Hub](https://aka.ms/openapiportal) was used for checking validation status and next steps.
### ARM API Review Checklist
- [ ] Service team MUST add the "WaitForARMFeedback" label if the management plane API changes fall into one of the below categories. 
- adding/removing APIs.
- adding/removing properties.
- adding/removing API-version. 
- adding a new service in Azure.

Failure to comply may result in delays for manifest application. Note this does not apply to data plane APIs.
- [ ] If you are blocked on ARM review and want to get the PR merged urgently, please get the ARM oncall for reviews (RP Manifest Approvers team under Azure Resource Manager service) from IcM and reach out to them. 
Please follow the link to find more details on [API review process](https://armwiki.azurewebsites.net/rp_onboarding/ResourceProviderOnboardingAPIRevieworkflow.html).

### CODE REVIEW

File: specification/compute/resource-manager/Microsoft.Compute/stable/2019-07-01/compute.json [hunk 1]
```diff
@@ -8383,6 +8412,16 @@
         "managedDisk": {
           "description": "The managed disk parameters.",
           "$ref": "#/definitions/VirtualMachineScaleSetManagedDiskParameters"
+        },
+        "diskIOPSReadWrite": {
+          "type": "integer",
```
Comment (hyonholee) [inline specification/compute/resource-manager/Microsoft.Compute/stable/2019-07-01/compute.json]: Is this not 'readOnly' in VMSS?  Can a user set this value only for VMSS and not for VM?

File: specification/compute/resource-manager/Microsoft.Compute/stable/2019-07-01/compute.json [hunk 2]
```diff
@@ -7092,7 +7109,19 @@
...
           "description": "Specifies whether the data disk is in process of detachment from the VirtualMachine/VirtualMachineScaleset"
-        }
+        },
+        "diskIOPSReadWrite": {
+          "type": "integer",
+          "readOnly": true,          
+          "format": "int64",
+          "description": "Specifies the Read-Write IOPS for the managed disk when StorageAccountType is UltraSSD_LRS. Returned only for VirtualMachine ScaleSet VM disks. Can be updated only via updates to the VirtualMachine Scale Set."
+        },
+        "diskMBpsReadWrite": {
```
Comment (hyonholee) [inline specification/compute/resource-manager/Microsoft.Compute/stable/2019-07-01/compute.json]: Is this updatable?

File: specification/compute/resource-manager/Microsoft.Compute/stable/2019-07-01/compute.json [hunk 3]
```diff
@@ -8383,6 +8412,16 @@
         "managedDisk": {
           "description": "The managed disk parameters.",
           "$ref": "#/definitions/VirtualMachineScaleSetManagedDiskParameters"
+        },
+        "diskIOPSReadWrite": {
```
Comment (hyonholee) [inline specification/compute/resource-manager/Microsoft.Compute/stable/2019-07-01/compute.json]: Is this updatable with PATCH or PUT?

File: specification/compute/resource-manager/Microsoft.Compute/stable/2019-07-01/compute.json (hunk 2 above)
Comment (robhat1) [reply specification/compute/resource-manager/Microsoft.Compute/stable/2019-07-01/compute.json]: At a VMSS level, yes.

File: specification/compute/resource-manager/Microsoft.Compute/stable/2019-07-01/compute.json (hunk 3 above)
Comment (robhat1) [reply specification/compute/resource-manager/Microsoft.Compute/stable/2019-07-01/compute.json]: Yes

**Pull request merged**